### PR TITLE
add getApp Details to Passage class

### PR DESCRIPTION
**Pull request by himichaelroberts**

### CODE REVIEW

File: src/classes/Passage.ts [hunk 1]
```diff
@@ -272,4 +273,32 @@ export default class Passage {
...
+
+    /**
+   * Get App Info about an app
+   *
+   * @return {Promise<AppObject>} Passage App object
+   */
+    async getApp(): Promise<AppObject> {
+        if (!this.#apiKey) {
+            throw new Error("A Passage API key is needed to make a getUser request");
+        }
```
Comment (dylanbrookes) [inline src/classes/Passage.ts:285]: Is there a reason we're requiring an API key to make this request? The endpoint `https://api.passage.id/v1/apps/{appID}` doesn't require authentication / an authorization header

File: src/classes/Passage.ts [hunk 2]
```diff
@@ -272,4 +273,32 @@ export default class Passage {
...
+
+        const appData: AppObject = await axios
+            .get(`https://api.passage.id/v1/apps/${this.appID}`, {
+                headers: {
+                    Authorization: `Bearer ${this.#apiKey}`,
+                },
+            })
```
Comment (dylanbrookes) [inline src/classes/Passage.ts:292]: We likely don't need this authorization header since this is an unauthenticated route

**Review by dylanbrookes — CHANGES_REQUESTED** [review]

**Comment by dylanbrookes** [comment]
Looks good - just the one small thing about getting app details without needing the authorization header. Also, thoughts on adding to the readme for the new getApp() method?

**Comment by calhorn95** [comment]
@himichaelroberts can you also add documentation for this new function to the README.  Thank you

**Pull request merged**